services/src/main/java/org/keycloak/authentication/requiredactions/VerifyUserProfile.java

```diff
@@ -46,11 +41,15 @@
 /**
  * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
  */
-public class VerifyUserProfile implements RequiredActionProvider, RequiredActionFactory, DisplayTypeRequiredActionFactory {
+public class VerifyUserProfile extends UpdateProfile {
 
     @Override
     public InitiatedActionSupport initiatedActionSupport() {
-        return InitiatedActionSupport.SUPPORTED;
+        return InitiatedActionSupport.NOT_SUPPORTED;
```

Comment:
Why don't support AIA?

Reply:
I don't think we needs it as AIA. It triggers automatically, so client can call common auth flow, no need to ask for this action explicitly in the auth call request (if I understand it correctly, the AIA is one additional parameter in auth request to explicitly ask for some required action which don't trigger automatically)

Reply:
Not necessarily related to authentication. But originally targeted for allowing invoking specific actions from the account console (or other trusted clients).

I would revert this change. I need to also check (and write a test) to assert the behavior when running it as an AIA. I originally thought it makes sense but after the lastest changes we did I need to review this.